web/server/entrypoint.go

```diff
@@ -328,6 +329,7 @@ func (s *robotServer) serveWeb(ctx context.Context, cfg *config.Config) (err err
 				restartInterval = newRestartInterval
 
 				if mustRestart {
+					s.logger.Debugf("backtrace at robot restart, %s", string(debug.Stack()))
```

Comment:
should it be `infof`? otherwise we'd lose it if we aren't in debug mode

Reply:
Ah good catch Cheuk, thanks! I agree that this should be at the info level.

I imagine the concern is about being conservative on how much gets spit out to logs? I'd advocate for doing info for now and looping back with some more code if there are complaints.